server/services/IndexService.ts

```diff
@@ -175,11 +183,15 @@ export default class IndexService {
         body: {
           ok: true,
           response: {
-            indices: paginatedIndices.map((catIndex: CatIndex) => ({
-              ...catIndex,
-              managed: managedStatus[catIndex.index] ? "Yes" : "No",
-              managedPolicy: managedStatus[catIndex.index],
-            })),
+            indices: customSort(
```

Comment:
This sort on managed would only apply when the sortField is chosen to be managed as I understand.

Reply:
Hello @bowenlan-amzn, thank you for bringing this to my attention. I noticed that the code was displaying indices based on the "managed" field, regardless of whether the sort field was "managed" or not. I have updated the logic, and it is now functioning as expected. Could you please take a look?